discovery_child_development/pipeline/openalex_metaflow.py

```diff
@@ -0,0 +1,168 @@
+"""
+works pipeline
+--------------
+
+A pipeline that takes a list of concept IDs and years, and outputs OpenAlex API results.
+
+The thought behind this is to break the results into manageable yearly chunks. For a given year
+and high level concept, the output works may be well over 2GB in size when saved to json.
+"""
+import itertools
+import json
+import requests
+from metaflow import FlowSpec, S3, step, Parameter, retry, batch
+import os
+import boto3
+from dotenv import load_dotenv
+
+# Amend this to your desired concepts/years. OpenAlex allows up to 50 parameters
+# per query, so code is included by default to chunk up the concepts into 40s.
+CONCEPT_IDS = [
+    "C109260823",  # child development
+    "C2993937534",  # childhood development
+    "C2777082460",  # early childhood
+    "C2911196330",  # child rearing
+    "C2993037610",  # child care
+    "C2779415726",  # child protection
+    "C2781192327",  # child behavior checklist
+    "C15471489",  # child psychotherapy
+    "C178229462",  # early childhood education
+    # "C138496976",  # developmental psychology (level 1).
+]
+
+YEARS = [2023]
```

Comment:
Just so I understand correctly - so far, we've only looked at and clustered data from 2023, yes?

Reply:
Yes exactly